scripts/install/sonarr.sh

```diff
@@ -82,21 +90,21 @@ _sonarrold_flow() {
 
         mkdir -p /root/swizzin/backups/
         echo_progress_start "Copying files to a backup location"
-        cp -R /home/"${sonarroldowner}"/.config/NzbDrone /root/swizzin/backups/sonarrold.bak
+        cp -R /home/"${SONARR_OLD_OWNER}"/.config/NzbDrone /root/swizzin/backups/sonarrold.bak
         echo_progress_done "Backups copied"
 
-        if [[ -d /home/"${sonarrv3owner}"/.config/sonarr ]]; then
-            if ask "$sonarrv3owner already has a sonarrv3 directory. Overwrite?" Y; then
+        if [[ -d /home/"${user}"/.config/sonarr ]]; then
+            if ask "$user already has a sonarrv3 directory. Overwrite?" Y; then
                 rm -rf
-                cp -R /home/"${sonarroldowner}"/.config/NzbDrone /home/"${sonarrv3owner}"/.config/sonarr
+                cp -R /home/"${SONARR_OLD_OWNER}"/.config/NzbDrone /home/"${user}"/.config/sonarr
             else
                 echo_info "Leaving v3 dir as is, why did we do any of this..."
             fi
         else
-            cp -R /home/"${sonarroldowner}"/.config/NzbDrone /home/"${sonarrv3owner}"/.config/sonarr
+            cp -R /home/"${SONARR_OLD_OWNER}"/.config/NzbDrone /home/"${user}"/.config/sonarr
         fi
 
-        systemctl stop sonarr@"${sonarroldowner}"
+        systemctl stop sonarr@"${SONARR_OLD_OWNER}"
```

Comment:
if sonarr v2 is being upgraded to v3 then v2 should stay running; the package will handle the migrations.

Reply:
It sadly will not as the old install was running in `screen`, which the upgrader was not designed to handle. More info: https://discord.com/channels/383686866005917708/383688189941907459/727264260585160875